common/scripted_triggers/wc_war_triggers.txt

```diff
@@ -1,7 +1,20 @@
-﻿has_sea_invader_innovation = {
+﻿culture_can_raid_at_sea_even_if_feudal_trigger = {
 	OR = {
 		has_innovation = innovation_longboats
-		has_cultural_parameter = culture_can_raid_at_sea_even_if_feudal
+		prefers_piracy_trigger = yes
```

Comment:
I said I would remove this from the trigger. The cultures listed in this list will have sea raiding by default.
There are 2 reasons:
1) This feature isn't mentioned in the tradition or culture descriptions thus the players aren't informed.
2) They pick the piracy tradition and get nothing from it.